source/_integrations/spotify.markdown

```diff
@@ -36,9 +36,10 @@ an application at Spotify using the Spotify Developer website. This will
 provide you with the Spotify application credentials Home Assistant needs
 to allow you to log in with your Spotify account.
 
-1. Log in to the [Spotify Developer](https://developer.spotify.com) Dashboard.
-2. Visit the [My Applications](https://developer.spotify.com/my-applications/#!/applications) page.
-3. Click the **CREATE AN APP** button in the top right. Enter a name and
+1. If Spotify was previously integrated in your Home Assistant with a _different_ account, remove the previous Spotify account credentials via https://my.home-assistant.io/redirect/application_credentials/.
```

Comment:
I know what you might try to say I think, however, it is partly incorrect/misleading. For example: One doesn't need to delete/remove other credentials, as you could integrate multiple application credentials on a single Home Assistant instance.

Please also, don't use bare URLs.